Scoop/NetworkManager.swift

```diff
@@ -173,6 +173,25 @@ class NetworkManager {
             }
         }
     }
+
+    func getBlockedUsers(completion: @escaping (Result<[BaseUser], Error>) -> Void) {
+        AF.request("\(hostEndpoint)/api/block/", method: .get, encoding: JSONEncoding.default, headers: headers).validate().responseData { response in
+            switch response.result {
+            case .success(let data):
+                let jsonDecoder = JSONDecoder()
+                jsonDecoder.keyDecodingStrategy = .convertFromSnakeCase
+                do {
+                    let blockedUsers = try jsonDecoder.decode([BaseUser].self, from: data)
+                    completion(.success(blockedUsers))
+                } catch {
+                    completion(.failure(error))
+                }
+            case .failure(let error):
+                completion(.failure(error))
+                print("Request getBlockedUsers Failed: \(error.localizedDescription)")
+            }
```

Comment:
I'm not a fan of how Scooped does networking. There are too many unnecessary print statements as well as inconsistencies with them. Logging is a powerful tool and should not be abused. I would note this in the handoff doc.